doc/payum-example/README.md

```diff
@@ -0,0 +1,61 @@
+## Abstract
```

Comment:
Is it usefull to repeat the whole readme?
maybe the main readme could point to this one containing and explaining only the few files commited?

Reply:
Actually I was thinking the best thing to do where to define that there is two types of examples in the main readme, and then point to two others readme in each example that would explain their specifications.

Reply:
:+1:  but maybe it will require some refection in order to separate the general explication from the stripe implementation

Reply:
Yes, that's why I did separate the two readme in the first place.